script.js

```diff
@@ -0,0 +1,54 @@
+
+let slidePosition = 0;
+const slides = document.getElementsByClassName('carousel-item');
```

Comment:
For styling use class names, but for JavaScript use id to get elements

Reply:
I used GetElementByClassName because I needed to return all items with this class name, while GetElementById selects only one(unique) element from html.

Reply:
I used GetElementByClassName because I needed to select all items with this class name, while GetElementById selects only one(unique) element from html.